test/functorch/test_aotdispatch.py

```diff
@@ -285,24 +287,20 @@ def is_in_base(t, maybe_tensors):
     return False
 
 
-def skipIfDynamoInput(reason, xfail=False):
+def skipIfDynamoInput(reason):
     """
     Skip TestAOTAutograd if running with dynamo input
     """
 
     def decorator(func):
         @wraps(func)
         def wrapper(self, *args, **kwargs):
-            fn = func
             if isinstance(self, TestAOTAutogradWithDynamo):
-                if xfail:
```

Comment:
The previous xfail actually wasn't doing what I want: in fact it was just skipping the test and passing it. Because in python 3 xfail doesn't work on functions but on test cases, you have to set it on the class before self is known (which is dumb because skipTest still works?). 

So I'm using xfail_inherited_tests to handle the xfail tests in the dynamo suite instead.